keadm/cmd/keadm/app/cmd/beta/init-beta.go

```diff
@@ -0,0 +1,183 @@
+
+keadm beta init --advertise-address=127.0.0.1 [--set cloudcore-tag=v1.9.0] --profile version=v1.9.0 -n kubeedge --kube-config=/root/.kube/config
+
+  - kube-config is the absolute path of kubeconfig which used to secure connectivity between cloudcore and kube-apiserver
+`
+)
+
+// NewCloudInit represents the keadm init command for cloud component
+func NewBetaInit() *cobra.Command {
+	BetaInit := newBetaInitOptions()
+
+	tools := make(map[string]types.ToolsInstaller)
+	flagVals := make(map[string]types.FlagData)
+
+	var cmd = &cobra.Command{
+		Use:     "init",
+		Short:   "Bootstraps cloud component. Checks and install (if required) the pre-requisites.",
+		Long:    cloudBetaInitLongDescription,
+		Example: cloudBetaInitExample,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			checkFlags := func(f *pflag.Flag) {
+				util.AddToolVals(f, flagVals)
+			}
+			cmd.Flags().VisitAll(checkFlags)
+			err := AddBetaInit2ToolsList(tools, flagVals, BetaInit)
+			if err != nil {
+				return err
+			}
+			return ExecuteBetaInit(tools)
+		},
+	}
+
+	addBetaInitJoinOtherFlags(cmd, BetaInit)
+	addHelmValueOptionsFlags(cmd, BetaInit)
+	addForceOptionsFlags(cmd, BetaInit)
+	return cmd
+}
+
+//newBetaInitOptions will initialise new instance of options everytime
+func newBetaInitOptions() *types.BetaInitOptions {
+	opts := &types.BetaInitOptions{}
+	opts.KubeConfig = types.DefaultKubeConfig
+	return opts
+}
+
+func addBetaInitJoinOtherFlags(cmd *cobra.Command, BetaInitOpts *types.BetaInitOptions) {
+	cmd.Flags().StringVar(&BetaInitOpts.AdvertiseAddress, types.AdvertiseAddress, BetaInitOpts.AdvertiseAddress,
+		"Use this key to set IPs in cloudcore's certificate SubAltNames field. eg: 10.10.102.78,10.10.102.79")
+
+	cmd.Flags().StringVar(&BetaInitOpts.KubeConfig, types.KubeConfig, BetaInitOpts.KubeConfig,
+		"Use this key to set kube-config path, eg: $HOME/.kube/config")
+
+	cmd.Flags().StringVar(&BetaInitOpts.Manifests, types.Manifests, BetaInitOpts.Manifests,
+		"Allow appending file directories of k8s resources to keadm, separated by commas")
+
+	cmd.Flags().StringVarP(&BetaInitOpts.Manifests, types.Files, "f", BetaInitOpts.Manifests,
+		"Allow appending file directories of k8s resources to keadm, separated by commas")
+
+	cmd.Flags().StringVarP(&BetaInitOpts.Namespace, types.Namespace, "n", BetaInitOpts.Namespace,
+		"Namespace to install kubeedge cloud component, default is kubeedge")
+
+	cmd.Flags().BoolVarP(&BetaInitOpts.DryRun, types.DryRun, "d", BetaInitOpts.DryRun,
+		"Print the generated k8s resources on the stdout, not actual excute. Always use in debug mode")
+
+	cmd.Flags().StringVar(&BetaInitOpts.CloudcoreImage, types.CloudcoreImage, BetaInitOpts.CloudcoreImage,
+		"The whole image of the cloudcore, default is kubeedge/cloudcore:v1.9.0")
+
+	cmd.Flags().StringVar(&BetaInitOpts.CloudcoreTag, types.CloudcoreTag, BetaInitOpts.CloudcoreTag,
+		"The image tag of the cloudcore, default is v1.9.0")
```

Comment:
CloudcoreImage is like `kubeedge/cloudcore`, without image tag,  and CloudcoreTag is like `v1.9.0`?

Reply:
yes, will fix